backend/dal/Repositories/OrganizationRepository.cs

```diff
@@ -93,10 +101,36 @@ public PimsOrganization Add(PimsOrganization organization, bool userOverride)
                 }
             }
 
-            var createdOrganization = this.Context.PimsOrganizations.Add(organization);
-            this.Context.CommitTransaction();
+            this.Context.PimsOrganizations.Add(organization);
+            return organization;
+        }
+
+        /// <summary>
+        /// Update the specified organization contact.
+        /// </summary>
+        /// <param name="organization"></param>
+        /// <returns></returns>
+        public PimsOrganization Update(PimsOrganization organization)
+        {
+            organization.ThrowIfNull(nameof(organization));
+            this.User.ThrowIfNotAuthorized(Permissions.ContactEdit);
+
+            var orgId = organization.Id;
+            var existingOrganization = this.Context.PimsOrganizations.Where(o => o.OrganizationId == orgId).FirstOrDefault()
+                 ?? throw new KeyNotFoundException();
+
+            // update main entity - PimsOrganization
+            Context.Entry(existingOrganization).CurrentValues.SetValues(organization);
+
+            // TODO: Verify these are working as expected!!!
```

Comment:
I will remove this TODO when the full end-to-end changes are implemented (including frontend forms)